cabal.project

```diff
@@ -12,10 +12,14 @@ repository cardano-haskell-packages
 
 -- See CONTRIBUTING for some Nix commands you will need to run if you
 -- update either of these.
--- Bump this if you need newer packages from Hackage
+-- NOTE: repeating the index-state for Hackage is a temporary workaround
```

Comment:
Yeah, this temporary will turn out to be permanent unless we check in some way that this problem has been fixed

Reply:
I might have been seduced by optimism here :)